documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploy Charmed MongoDB
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```shell
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```shell
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
```

Comment:
I got a bit confused here as well. I went away for a few hours while it was deploying (took a while on my slow internet). When I came back to it, I was pretty sure it was completed, but then I ran the "watch -c juju status --color" command anyway. I got the correct output according to what's here, but I was a bit confused about when and why I should use the command. Would I expect different output if it's not yet completed? Would I have to run it multiple times, or does it continuously update until reaching the output shown below?

(Also how do I get out of that output screen? enter, q, and Esc don't work - I found out by process of elimination that it's ctrl+c)